static/src/stylesheets/_vars.scss

```diff
@@ -114,6 +114,10 @@ $frontend-palette: (
     crosswordAccent1:       guss-colour(multimedia-support-1, $pasteup-palette),
     crosswordAccent2:       guss-colour(news-main-2, $pasteup-palette),
 
+    sudoku-accent-1:        guss-colour(multimedia-support-1, $pasteup-palette),
```

Comment:
Tell me which colour should be which and I'll change it. I don't understand your whacky design lingo!